### PR TITLE
[.NET] Flatten RichTextBlock object model

### LINKED ISSUE #2572: [1.2] .NET RichTextBlock OM is a bit odd
Is this really how we want the OM to look for richtextblock on .NET platforms?

```c#
                AdaptiveCard card = new AdaptiveCard("1.2")
                {
                    Body =
                    {
                        new AdaptiveTextBlock() { Text = "Hello World" },
                        new AdaptiveRichTextBlock()
                        {
                            Paragraphs =
                            {
                                new AdaptiveRichTextBlock.AdaptiveParagraph()
                                {
                                    Inlines =
                                    {
                                        new AdaptiveRichTextBlock.AdaptiveParagraph.AdaptiveTextRun()
                                        {
                                            Text = "this thing on?"
                                        }
                                    }
                                }
                            }
                        }
                    }
                };
```

### PR DESCRIPTION
**Pull request by paulcam206**
Fixes #2572.

### CODE REVIEW

File: source/dotnet/Library/AdaptiveCards/AdaptiveRichTextBlock.cs [hunk 1]
```diff
@@ -60,94 +60,98 @@ public AdaptiveRichTextBlock()
...
-        [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
-        public class AdaptiveParagraph
-        {
-            [JsonRequired]
-            [JsonConverter(typeof(IgnoreEmptyItemsConverter<AdaptiveTextRun>))]
+
+    }
+
+    [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
+    public class AdaptiveParagraph
```
Comment (RebeccaAnne) [inline source/dotnet/Library/AdaptiveCards/AdaptiveRichTextBlock.cs]: >AdaptiveParagraph [](start = 17, length = 17)

Should this go in it's own file? That's the convention we've been using in UWP.

Comment (paulcam206) [reply source/dotnet/Library/AdaptiveCards/AdaptiveRichTextBlock.cs]: sure thing :)

---
In reply to: [270478025](https://github.com/Microsoft/AdaptiveCards/pull/2581#discussion_r270478025) [](ancestors = 270478025)

File: source/dotnet/Library/AdaptiveCards/AdaptiveParagraph.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    public class AdaptiveParagraph
+    {
+        [JsonRequired]
+        [JsonConverter(typeof(IgnoreEmptyItemsConverter<AdaptiveTextRun>))]
+#if !NETSTANDARD1_3
+        [XmlElement(typeof(AdaptiveTextRun))]
+#endif
+        public List<AdaptiveTextRun> Inlines { get; set; } = new List<AdaptiveTextRun>();
+
+
```
Comment (RebeccaAnne) [inline source/dotnet/Library/AdaptiveCards/AdaptiveParagraph.cs]: [nit] Extra white space?

**Comment by paulcam206** [comment]
bypassed checks due to broken cla status

**Pull request merged**